docs/en/ingest-management/troubleshooting.asciidoc

```diff
@@ -161,6 +162,33 @@ property in the `kibana.yml` configuration file. For example:
 xpack.encryptedSavedObjects.encryptionKey: "something_at_least_32_characters"
 ----
 
+[discrete]
+[[process-not-root]]
+== {agent} fails on Windows with `Agent process is not root/admin or validation failed` message
+
+Make sure the user has administrator-level privileges.
+
+If you're using the {elastic-endpoint} integration, also make sure you're
+running {agent} under the SYSTEM account.
+
+To run {agent} under the SYSTEM account, you can:
```

Comment:
I figured it might be useful to show this but my windows skills are kind of rusty. Someone will need to confirm that this is correct. I did some light testing on a VM, and it seemed OK.

Reply:
I like what you have